portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,51 @@
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+    padding-top: 20px;
+    text-align: center;
+    position: static
+}
+#summary {
+    position: absolute;
+    left: 15px;
+    width:30%;
+}
+#picture {
+    position: absolute;
+    right: 15px;
+}
+
+html {
+    background: rgba(153, 153, 153, 0.15);
+}
+.button {
+  background-color:rgb(255,255,255); /* Green */
```

Comment:
Be consistent on spacing: `background-color: rgb(255, 255, 255);`

There are a couple other places in this file where you're missing the space after `:` too.